src/Breadcrumbs.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Honeybadger;
+
+use Honeybadger\Support\EvictingQueue;
+
+class Breadcrumbs extends EvictingQueue
+{
+    /**
+     * @param $item
+     *
+     * @return self
+     */
+    public function add($item)
+    {
+        $item = [
+            'message' => (string) $item['message'],
+            'category' => (string) ($item['category'] ?? 'custom'),
+            'metadata' => $this->sanitize($item['metadata'] ?? []),
+            'timestamp' => (int) ($item['timestamp'] ?? time()),
```

Comment:
I believe the timestamp should be iso8601 format, like this: "2021-03-10T23:12:51.748Z". I'm not sure if an integer will work. cc @stympy

Reply:
Silly me🤦‍♂️

Reply:
Was able to get ISO8601, but in this form `2021-03-12T17:28:36+00:00`. Getting the Zulu time version in PHP is a bit more involved. Will the API be fine with that?

Reply:
The form `2021-03-12T17:28:36+00:00` should be fine